src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -195,6 +195,8 @@ def __init__(
             data = data.clone()
         self.frame = data
 
+        self.frame = self.frame.fill_nan(None)
```

Comment:
why is this?

Reply:
there's a difference between nan values an null values in the apache arrow dataformat (which is the backbone of polars). nan values include +inf/-inf and by definition also don't compare equal to another.

with null values you mark these values as really missing and calculating mean values for example 

here's more info on that: https://docs.pola.rs/user-guide/expressions/missing-data/

Reply:
I added a comment to explain what this does